UefiTestingPkg/FunctionalSystemTests/MemoryProtectionTest/App/MemoryProtectionTestApp.c

```diff
@@ -41,10 +43,14 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include "../MemoryProtectionTestCommon.h"
 #include "UefiHardwareNxProtectionStub.h"
 
-#define UNIT_TEST_APP_NAME     "Heap Guard Test"
-#define UNIT_TEST_APP_VERSION  "0.5"
+#define UNIT_TEST_APP_NAME                 "Memory Protection Test"
+#define UNIT_TEST_APP_VERSION              "2.0"
+#define UNIT_TEST_WARM_RESET_STRING        L"--WarmReset"

```

Comment:
Unless there's a specific reason to mention the reset type, at the app parameter level I'd simply call this reset and let parameter documentation describe that a warm reset is currently performed.

Reply:
Changed from WarmReset to Reset